Formula/libpostal.rb

```diff
@@ -0,0 +1,26 @@
+class Libpostal < Formula
+  desc "C library for parsing/normalizing street addresses around the world"
+  homepage "https://github.com/openvenues/libpostal"
+  url "https://github.com/openvenues/libpostal/archive/v1.0.0.tar.gz"
+  sha256 "3035af7e15b2894069753975d953fa15a86d968103913dbf8ce4b8aa26231644"
+  head "https://github.com/openvenues/libpostal.git"
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+
+  def install
+    system "./bootstrap.sh"
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--datadir=#{prefix}/data",
+                          "--prefix=#{prefix}"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    system './libpostal "Quatre vingt douze Ave des Champs-Élysées"'
```

Comment:
Capture the output of the command with:

```
  output = shell_output(#{bin}/libpostal 'Quatre vingt douze Ave des Champs-Élysées'")
```

and then check it with `assert_match` or `assert_equal`